app/controllers/exam_templates_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ExamTemplatesController < ApplicationController
+
+  before_filter      :authorize_only_for_admin
+
+  layout 'assignment_content'
+
+  def index
+    @assignment = Assignment.find(params[:assignment_id])
+    @exam_templates = ExamTemplate.all
```

Comment:
You should only grab the exam templates associated with the particular assignment.